### DISCUSSION TITLE
Where's windeployqt (Qt6)?

### DISCUSSION THREAD

**Discussion by drfiemost** [msys2/MINGW-packages#17785] [Q&A] [answered]
Trying to use windeployqt.exe in a github workflow but it's not being found.
I'm using the following packages:
* mingw-w64-x86_64-qt6-base
* mingw-w64-x86_64-qt6-shadertools
* mingw-w64-x86_64-qt6-multimedia
* mingw-w64-x86_64-qt6-tools

Expected it to be in qt6-tools but it's not there:
https://packages.msys2.org/package/mingw-w64-x86_64-qt6-tools?repo=mingw64

Where is it gone?

**Comment by Biswa96** [comment, marked as answer]
The name was changed to windeployqt6.exe or windeployqt-qt6.exe to prevent file name conflict with qt5. For example, see the file list in qt6-base package https://packages.msys2.org/package/mingw-w64-x86_64-qt6-base?repo=mingw64

**Comment by drfiemost** [comment]
Ah, missed that! Thanks for the quick response.
